src/internal/packager/helm/repo.go

```diff
@@ -251,7 +252,7 @@ func (h *Helm) DownloadPublishedChart(ctx context.Context, cosignKeyPath string)
 	}
 
 	// Download the file into a temp directory since we don't control what name helm creates here
-	temp := filepath.Join(h.chartPath, "temp")
```

Comment:
Noticed that we don't have to reach into the `helm` struct to access this chartPath. It's just in lexical scope. I just think that's neat. 😄